rag/flow/compiler/compiler.py

```
@@ -154,24 +259,36 @@ async def _invoke(self, **kwargs):
             return
         active_templates = filtered_templates
 
+        if self._canvas._kb_id:
+            e, kb = KnowledgebaseService.get_by_id(self._canvas._kb_id)
```

Ruff (0.15.20)

[warning] Unpacked variable `e` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)